### ISSUE TITLE
cardano-node error

### ISSUE THREAD

**Issue by lijianl** [cardano-foundation/cardano-wallet#1988]
my cardano-wallet met a error:
`[nuls-nov:cardano.node.DnsSubscription:Error:13712] [2020-07-31 15:04:23.86 UTC] [String "Application Exception: 127.0.0.1:3001 MuxError MuxBearerClosed \"<socket: 28> closed when reading data, waiting on next header True\" []",String "SubscriptionTrace",String "\"relays-new.cardano-mainnet.iohk.io\""]
[nuls-nov:cardano.node.ErrorPolicy:Warning:63] [2020-07-31 15:04:23.86 UTC] [String "ErrorPolicySuspendPeer (Just (ApplicationExceptionTrace (MuxError MuxBearerClosed \"<socket: 28> closed when reading data, waiting on next header True\" []))) 20s 20s",String "ErrorPolicyTrace",String "127.0.0.1:3001"]
[nuls-nov:cardano.node.DnsSubscription:Notice:13730] [2020-07-31 15:04:32.79 UTC] [String "Connection Attempt Start, destination 127.0.0.1:3001",String "SubscriptionTrace",String "\"relays-new.cardano-mainnet.iohk.io\""]
[nuls-nov:cardano.node.DnsSubscription:Notice:13731] [2020-07-31 15:04:32.82 UTC] [String "Connection Attempt Start, destination 18.140.13.156:3001",String "SubscriptionTrace",String "\"relays-new.cardano-mainnet.iohk.io\""]
[nuls-nov:cardano.node.DnsSubscription:Notice:13731] [2020-07-31 15:04:32.82 UTC] [String "Connection Attempt End, destination 127.0.0.1:3001 outcome: ConnectSuccessLast",String "SubscriptionTrace",String "\"relays-new.cardano-mainnet.iohk.io\""]
[nuls-nov:cardano.node.ErrorPolicy:Notice:63] [2020-07-31 15:04:32.82 UTC] [String "ErrorPolicySuspendConsumer (Just (ConnectionExceptionTrace (SubscriberError {seType = SubscriberParrallelConnectionCancelled, seMessage = \"Parrallel connection cancelled\", seStack = []}))) 1s",String "ErrorPolicyTrace",String "127.0.0.1:3001"]`

there are few docs to check the error?  and my cardano-wallet-byron can not work normally by now ?  could you help me with this

**Comment by lijianl** [comment]
`nuls-nov:cardano.node.LocalErrorPolicy:Error:64] [2020-08-01 07:53:17.85 UTC] [String "ErrorPolicyUnhandledApplicationException (MuxError MuxBearerClosed \"<socket: 359> closed when reading data, waiting on next header True\")",String "ErrorPolicyTrace",String "LocalAddress {getFilePath = \"\"}"]`


cardano-node 1.18.0 error...... how to solve the problem

**Comment by rvl** [comment]
Hi @lijianl,

1. Use [`cardano-wallet-shelley`](https://github.com/input-output-hk/cardano-wallet/releases) now because we have hard forked into the shelley era.
2. The log message from cardano-node `MuxBearerClosed ... waiting on next header True` means that a connection to cardano-node was closed by the other end. It is actually not an error, even the the log message has Error severity. I have raised this with the networking team.
